src/rules/sort-collections.ts

```diff
@@ -60,7 +60,7 @@ export const rule = createRule<Options>({
 				const currentOrder = collection.properties;
 				let desiredOrder: JsonAST.JSONProperty[];
 
-				// If it's any property other than `scripts`, simply sort lexicographically
+				// If it's any property other than `scripts`, simply sort lexically
```

Comment:
"Lexicographically" was probably the better word to use (here and in the violation messages).  Curious what what prompted the change to "lexically"?  https://en.wikipedia.org/wiki/Lexicographic_order

Reply:
I'm not a native speaker, and at first I thought "lexically" and "lexicographically" meant the same in this context. I preferred the shorter form for consistency. After looking it up, I understand now that "lexicographically" is the more precise and correct term here. So, I think we should revert that.

Reply:
But today I can not do it.

Reply:
Sounds good!  I'll mark as Change Requested til you're able to get back to it.